added the option to checkGeneSymbols by using chromosome number

Aggarwal-Ayush opened the pull request:
Updated getCurrentMaps to include chromosome number in the human map and the option to checkGeneSymbols by using chromosome numbers.  NA is returned if the chromosome number is incorrect. hgnc.table.rda needs to be updated to cuurentHumanMap to incorporate this change.

lwaldron reviewed (changes requested):
Thank you @Aggarwal-Ayush, this is great! Any chance you could add unit tests to check it is working as you expect, as well?

Aggarwal-Ayush commented:
> 
> 
> Thank you @Aggarwal-Ayush, this is great! Any chance you could add unit tests to check it is working as you expect, as well?

Sure, I'll add the unit tests.

Aggarwal-Ayush commented:
Hi @lwaldron, I've added the unit tests as comments in the checkGeneSymbols.R file. It's working as I expected with the map specified as currentHumanMap. Once the hgnc.table.rda is updated to include chromosome numbers, it will work without having to specify map for human. 
Also, I've updated the function to return the same output as before but with a warning if the chromosome numbers are wrong, instead of returning NA.
